lib/visualization/draw_event_stream.py

```diff
@@ -302,15 +342,25 @@ def plot_between_frames(xs, ys, ts, ps, frames, frame_event_idx, args, plttype='
         img_ts = []
         for f_idx in frame_indices:
             img_ts.append(ts[f_idx[1]])
-        fname = os.path.join(args.output_path, "events_{:09d}.png".format(i))
+        fname_with_frame = os.path.join(args.output_path, "event_with_frame_{:09d}.png".format(i))
+        fname_with_event = os.path.join(args.output_path, "event_{:09d}.png".format(i))
```

Comment:
calling `plot_events` twice to create event-with-eventimage and event-with-grayframe.